xml/System.Security.Cryptography/SHA256.xml

```diff
@@ -171,6 +157,20 @@ The .NET Framework includes the implementations and their associated hashName va
 
  ]]></format>
         </remarks>
+        <exception cref="T:System.Reflection.TargetInvocationException">On the .NET Framework 4.6.1 and earlier versions only: The algorithm was used with Federal Information Processing Standards (FIPS) mode enabled, but is not FIPS compatible.</exception>
```

Comment:
Any FIPS related exception is a function of the input value passed in.  So this exception doesn't (easily) apply to this overload.

Reply:
But it does apply to the other one

Reply:
@bartonjs Oops, I didn't view your comment before [my last commit](https://github.com/dotnet/docs/pull/3636/commits/83f85e8cafcf3e209c8742cf26f1a9b5ed5897c0). What would be your recommendation regarding this Exception description ? Remove it (? Remove the mention to FIPS ?

Reply:
Your comment didn't appear in the Conversations tab when I reviewed the PR before merging, @bartonjs, so I completely missed it. I can open a separate PR to address the issue. But I'm not quite sure what to do -- like @sputier, I'm not sure whether the exception should be removed, whether it should be qualified in terms of .NET Framework versions that can throw the exception, or whether the text should just be reverted to its pre-PR state.

Reply:
Looking again at the method, it mentions that it throws if the specific value isn't supported by the FIPS mode.  That's still true on .NET Framework if someone explicitly requests SHA256Managed.  What we fixed in 4.6.2 was that the default lookup value would map to a FIPS-compatible one.

So the exception text about `Create(string)` could be deleted, or just made to be a .NET Framework-only comment.

Reply:
@bartonjs, OK. I'll open a PR to change the exception message to be for .NET Framework only, and to throw only when SHA256Managed is requested and FIPS mode is enabled.